### ISSUE TITLE
[BUG] [SM-J701MT] CRONOSKERNEL 5.0 ON ANDROID 10 AOSP

### ISSUE THREAD

**Issue by ghost** [ananjaser1211/Helios_7870#6]
I know, this kernel should only be used on Android Pie.
But I works fine on Android 10! Is just a little bug. (Maybe?)

I'm just facing random reboots every 20 minutes.

See:
https://github.com/phhusson/treble_experimentations/issues/1088

And see this for logcat:
[BugReport.zip](https://github.com/ananjaser1211/Helios_7870/files/4133688/BugReport.zip)

**Comment by Jannled** [comment]
I also encountered this issue on my J5 (2017) with latest Bootloader and Modem.

Since both the [Ananjaser own Cronos Kernel](https://forum.xda-developers.com/galaxy-j5/samsung-galaxy-j5-pro-2017-roms-kernels-recoveries--other-development/kernel-refined-kernel-v1-0-t3675500) and [Spartanicus Kraken Kernel](https://forum.xda-developers.com/galaxy-j7-2017/development/kernel-project-butter-t3957185) are based upon this work, this is the only solution if you would like to get Android 10 running on an Exynos7870 based device.

I tried both Kraken and Cronos, but the Kernel is persistently panicing after around 1455(+-5) seconds because the kernel has locked up. 
I had some runs with different ROM/Kernel combinations (all of them beeing Anrdoid 10), who needed longer to crash, but usually it takes the above mentioned time. (Android 9 is working for me without this issue)

In my logs the Kernel got stuck either at physical address [0x00000000bab4c000](https://github.com/ananjaser1211/Helios_7870/files/4498828/last_kmsg.13.37.txt) (like in @Noryub report) or [0x00000000bab84000](https://github.com/ananjaser1211/Helios_7870/files/4498829/last_kmsg.14.03.txt).

I currently don't have the knowledge to debug a kernel, so I hope that @ananjaser1211 is revisiting one of his old projects. If necessary, I will happily provide more debug information.
